### PR TITLE
CI: update check-labels-action

### PR DESCRIPTION
**Pull request by kaspar030**
### Contribution description

This PR bumps the check-labels-action to the new version that sets a "Process: missing approvals" label, to be used as bors "block_labels" instead of relying on bors to pick up changes to the check-labels status.

~~TESTING STILL~~ waiting for https://github.com/RIOT-OS/check-labels-action/pull/1 to be merged & tagged

### CODE REVIEW

**Comment by kaspar030** [comment]
> Is that label set / removed automatically?

That's the idea, I'm trying to [add that to the label checker](https://github.com/RIOT-OS/check-labels-action/pull/1).

**Comment by miri64** [comment]
As discussed offline: Let's merge this and test in master. `GITHUB_TOKEN` has restricted permissions for PRs (which was already a problem in the past). @kaspar030 please squash.

**Comment by miri64** [comment]
https://github.com/RIOT-OS/check-labels-action/pull/1 is merged. Please rebase and mark ready for review, so we can tentatively merge this (otherwise AFAIK there is no real way to test this)

File: .github/workflows/check-labels.yml [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+  pull_request_target:
     types: [opened, reopened, labeled, unlabeled, synchronize]
   pull_request_review:
     types: [submitted, dismissed]
 jobs:
   check-labels:
     runs-on: ubuntu-latest
     steps:
-    - uses: RIOT-OS/check-labels-action@v1.0.0
+    - uses: RIOT-OS/check-labels-action@master
```
Comment (kaspar030) [inline .github/workflows/check-labels.yml:11]: using a branch here should work, right?

**Comment by kaspar030** [comment]
If we merge this and sth goes wrong, the action might never succeed and we need to force merging any fix.
Should we maybe disable bors' check for the action until we know it works?

File: .github/workflows/check-labels.yml (hunk 1 above)
Comment (miri64) [reply .github/workflows/check-labels.yml:11]: Yepp (but we can also add a new version)

**Comment by miri64** [comment]
> Should we maybe disable bors' check for the action until we know it works?

Which check exactly?

**Comment by kaspar030** [comment]
> Which check exactly?

I mean this one:

https://github.com/RIOT-OS/RIOT/blob/6fb340d654ac8da07759cb9199c6aaa478589aa7/bors.toml#L6

File: .github/workflows/check-labels.yml (hunk 1 above)
Comment (kaspar030) [reply .github/workflows/check-labels.yml:11]: Let's see if this works at all or if the `pull_request_event` event is also forced to read-only permissions?

**Comment by miri64** [comment]
:+1: I was not sure if you meant disable bors altogether, but temporarily disabling the label check is fine for me. Can you piggyback this?

File: .github/workflows/check-labels.yml (hunk 1 above)
Comment (miri64) [reply .github/workflows/check-labels.yml:11]: I added a v1.1.0 tag now, btw.

Comment (kaspar030) [reply .github/workflows/check-labels.yml]: Ok, changed accordingly.

File: bors.toml [hunk 2]
```diff
@@ -3,7 +3,7 @@
 pr_status = [
  "python-tests",
  "tools-build-success",
- "check-labels",
+# "check-labels",
```
Comment (miri64) [inline bors.toml:6]: Care to add a comment why this was disabled? (just in case reverting this takes longer than expected again..)

**Review by miri64 — APPROVED** [review]
bors merge

**Comment by miri64** [comment]
bors merge ...

**Pull request merged**